src/providers/steem/auth.js

```diff
@@ -33,23 +32,23 @@ export const login = async (username, password) => {
       new Error('You are already logged in, please try to add another account'),
     );
   }
-
   // Public keys of user
   const publicKeys = {
-    active: account.active.key_auths.map(x => x[0]),
-    memo: account.memo_key,
-    owner: account.owner.key_auths.map(x => x[0]),
-    posting: account.posting.key_auths.map(x => x[0]),
+    activeKey: account.active.key_auths.map(x => x[0])[0],
```

Comment:
hmm have you tested this with different accounts? Because some accounts have multiple key_auths and index [0][0] might catch different publickey. for example: https://steemd.com/@hellosteem